### PR TITLE
Kuanysh12

### PR DESCRIPTION
**Pull request by Suomiprogramming**
Controller for fire department with the appropriate services for handling with the creation the new fire incident reports and viewing of them were added. Also, AutoMapper profiles and setting the default incident date were added.

### CODE REVIEW

File: UtopiaCity/Controllers/FireDepartment/FireDepartmentController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        public IActionResult Index()
+        {
+            var viewModel = _fireIncidentReportService.GetAllFireIncidentReportViewModels();
+            return View(viewModel);
+        }
+
+        [HttpGet]
+        public IActionResult CreateFireIncidentReport()
+        {
+            CreateFireIncidentReportViewModel viewModel = new CreateFireIncidentReportViewModel();
```
Comment (crocomoth) [inline UtopiaCity/Controllers/FireDepartment/FireDepartmentController.cs:26]: var is enough - it is obvious which type variable is

File: UtopiaCity/Controllers/FireDepartment/FireDepartmentController.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        public IActionResult CreateFireIncidentReport(CreateFireIncidentReportViewModel viewModel)
+        {
+            try
+            {
+                _fireIncidentReportService.CreateFireIncidentReport(viewModel);
+                return RedirectToAction("Index");
+            }
+            catch (Exception)
+            {
+
```
Comment (crocomoth) [inline UtopiaCity/Controllers/FireDepartment/FireDepartmentController.cs:40]: if exception is not hanlded properly, why "handle" it in first place?

File: UtopiaCity/Extensions/ServiceAdder.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using UtopiaCity.Services.FireDepartment.FireIncidentReportServices;
+
+namespace UtopiaCity.Extensions
+{
+    public static class ServiceAdder
```
Comment (crocomoth) [inline UtopiaCity/Extensions/ServiceAdder.cs:6]: when this pattern is used it is usually used like this - 1 class - 1 logical area - basically, one extension class per 1 person in our case, lets discuss this on wednesday

Comment (crocomoth) [inline UtopiaCity/Extensions/ServiceAdder.cs:6]: when this pattern is used it is usually used like this - 1 class - 1 logical area - basically, one extension class per 1 person in our case, lets discuss this on Wednesday

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/Controllers/FireDepartment/FireDepartmentController.cs (hunk 2 above)
Comment (Suomiprogramming) [reply UtopiaCity/Controllers/FireDepartment/FireDepartmentController.cs:40]: oh, sorry. I've forgotten to add return Bad Request

**Review by Usylli — APPROVED** [review]

**Pull request merged**